docs/2 Architecture/3 Standard Modules.md

```diff
@@ -58,10 +67,14 @@ _Read the [specification](https://github.com/cosmos/cosmos-sdk/blob/master/x/par
 
 Those modules are typically installed on default when using starport. There are a range of modules that are also part of the Cosmos SDK, additionally some other public modules have already reached a major level of usage and acceptance. We will look at more advanced modules in the next Chapter.
 
-## Summary
+## Using modules
+
+With starport you can add a module with the command `starport add modulename`. When adding a module manually to a blockchain application, it requires to edit the `app/app.go` and the `myappcli/main.go` with the according entries. Starport manages the code edits and additions for you conviniently.
```

Comment:
The command has changed, now it:
`starport module import` to import
`starport module create` for a new empty project